### PR TITLE
Update Web3 to Latest Stable (1.2.1)

### PR DESCRIPTION
**Pull request by edsonayllon**
Changes made were addition of `test/integration/targets/web3-1.0.0/module.d.ts`, updated web3 to web3@1.2.0, and changed `new Web3()` provider syntax.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ test/integration/targets/truffle/build
 test/integration/targets/web3-1.0.0/types/web3-contracts
 test/integration/targets/ethers/types/ethers-contracts
 test/package-test/types
+.DS_Store
```
Comment (krzkaczor) [inline .gitignore]: this should go to your global git ignore. No need to add it here.

File: test/integration/targets/web3-1.0.0/yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,2676 @@
+Arguments:
```
Comment (krzkaczor) [inline test/integration/targets/web3-1.0.0/yarn-error.log]: this file should not be committed

**Comment by edsonayllon** [comment]
@krzkaczor Made those changes

File: test/integration/targets/web3-1.0.0/yarn-error.log (hunk 2 above)
Comment (edsonayllon) [reply test/integration/targets/web3-1.0.0/yarn-error.log]: 42b716f

File: .gitignore (hunk 1 above)
Comment (edsonayllon) [reply .gitignore]: 42b716f

**Comment by eswarasai** [comment]
@krzkaczor - Any ETA on when this can be released? Thanks!

**Comment by krzkaczor** [comment]
I am still not possible to review this @edsonayllon . Look at the diff please.

1. Remove any changes to file permissons
2. Remove any changes introducing/removing only whitecharacters `.gitignore`

**Comment by edsonayllon** [comment]
https://github.com/ethereum-ts/TypeChain/pull/166

**Pull request closed without merging**